fonts/fonts.css

```diff
@@ -0,0 +1,8 @@
+  @font-face {
+      font-family: 'Montserrat';
+      src: url('montserrat-regular-webfont.woff2') format('woff2'),
+           url('montserrat-regular-webfont.woff') format('woff');
+      font-weight: normal;
+      font-style: normal;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/max_osipov/pullRequest?prid=5575857)